Week3/app/src/main/res/menu/main_menu.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Excellent job on the menu and very creative idea adding a choice between the different programming languages.